huaweicloud/config/config.go

```diff
@@ -179,12 +180,16 @@ func (c *Config) ObjectStorageClientWithSignature(region string) (*obs.ObsClient
 		}
 	}
 
+	httpClient := c.DomainClient.HTTPClient
+	clientConfigure := obs.WithHttpClient(&httpClient)
+	userAgentConfigure := obs.WithUserAgent(userAgent)
```

Comment:
custom defined user agent should also be supported

Reply:
done, customUserAgent has been supported